salt/states/kapacitor.py

```diff
@@ -47,6 +50,11 @@ def task_present(name,
     task_type
         Task type. Defaults to 'stream'
 
+    dbrps
+        A list of databases and retention policies in "dbname"."rpname" format
+        to fetch data from. For backward compatibility, the value of
+        'database' and 'retention_policy' will be merged as part of dbrps.
+
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag to these docs (and do the same for the docs in the module)?

Reply:
Sure, but how I can add it for mixed sections like this:

```
'''
Kapacitor state module.

:configuration: This module accepts connection configuration details either as
    parameters or as configuration settings in /etc/salt/minion on the relevant
    minions::

        kapacitor.unsafe_ssl: 'false'
        kapacitor.protocol: 'http'
        kapacitor.host: 'localhost'
        kapacitor.port: 9092

    This data can also be passed into pillar. Options passed into opts will
    overwrite options passed into pillar.

.. versionadded:: 2016.11.0
'''
```

Since `kapacitor.unsafe_ssl` and `kapacitor.protocol` are new, but rest are old (`2016.11.0`)